docs/README.md

```diff
@@ -2,22 +2,20 @@
 
 Welcome to the Serverless v1.0 documentation.
 
-Here you'll find all the necessary information you need to learn and understand Serverless.
-It'll show you how you can build next generation Serverless applications. Furthermore we'll deep dive into the
-internals of Serverless so that you know how it works and how you can extend and modify it!
-
 ## Quick start
 
-Follow these simple steps to install the beta, create and deploy your first service, run your function and remove the
-service afterwards.
+Follow these simple steps to install the beta, create and deploy your first service, run your function and remove the service afterwards.
 
-1. `npm install -g serverless@beta`
+1. `npm install -g serverless`
 2. `mkdir my-first-service && cd my-first-service`
```

Comment:
Could be updated once https://github.com/serverless/serverless/pull/1937 is merged (the path option will automatically create a directory).